### DISCUSSION TITLE
Firefly app container goes unhealthy

### DISCUSSION THREAD

**Discussion by Elvin-Arrow** [firefly-iii/firefly-iii#8599] [General]
Hi, I am trying to run firefly via docker compose using the files as provided in the installation instructions on the firefly documentation page. 

Running the `docker compose up -d` command starts the containers but soon after, the app container starts to show status as unhealthy. The container logs show health check as failing as can be seen in the logs attached below but I am not sure as to what is causing it to fail.

[docker-compose.log](https://github.com/firefly-iii/firefly-iii/files/14453210/docker-compose.log)

I am running this on Fedora Server 39. SELinux is enforced, not sure if that is causing this issue. I don't see any message saying the access was blocked though.

I have tried running the exact same set of files on a Ubuntu server instance and that works flawlessly.

Again I am using compose and env files as available on GitHub with no changes other than what the installation steps ask you to do.

**Comment by JC5** [comment]
I see a json error for composer.json which is very weird. Could you please share your docker compose file?

**Comment by Elvin-Arrow** [comment]
Sure, here is the compose file.

https://gist.github.com/Elvin-Arrow/064afb521c61003ed77956e9e02ea836

I downloaded this file from the following link using curl:

https://raw.githubusercontent.com/firefly-iii/docker/main/docker-compose.yml

Exact command used to download the file was:

```bash
curl -L -o docker-compose.yml https://raw.githubusercontent.com/firefly-iii/docker/main/docker-compose.yml
```

**Comment by JC5** [comment]
Strange, no clue. Perhaps delete the layers and retry?

**Comment by Elvin-Arrow** [comment]
Thanks for the suggestion. Here's what I did to fix the issue:

- Stopped all containers
- Stopped docker service
- Removed `/var/lib/docker` directory
- Started docker service

Ran the compose file again and that did the trick!
